content/platform/_index.md

```diff
@@ -53,46 +72,60 @@ sections:
 
         {{< video src="videos/jupyterlab.mp4">}}
 
+        <p class="figcaption">
+          Powered by <a href="https://jupyterlab.readthedocs.io"><img src="/images/logos/project/jupyterlab.svg" /></a> and <a href="https://mystmd.org"><img src="/images/logos/project/myst.svg" /></a> 
+        </p>
+
   - block: markdown
     id: environment-choice
     design:
         columns: 2
     content:
-      title: Choose a community environment, or create your own.
+      title: Choose a community environment, or create your own
       subtitle:  Community leaders can offer many environments for users to fit all of their workflows.
       text: |
 
         {{< figure src="/images/service/server-choices.png" >}}
 
+        <p class="figcaption">
+          Powered by <a href="https://jupyterhub.readthedocs.io"><img src="/images/logos/project/jupyterhub.svg" /></a> and <a href="https://repo2docker.readthedocs.io"><img src="/images/logos/project/repo2docker.png" /></a>
+        </p>
   - block: markdown
     id: knowledge-base
     design:
         columns: 2
         css_class: reverse-markdown-row
     content:
-      title: A shared knowledge base that connects with your hub.
+      title: A shared knowledge base that connects with your hub
       subtitle:  Shared knowledge bases allow communities to contribute their ideas and work to a shared space that is accessible to the community.
       text: |
 
         {{< video src="videos/jupyterbook.mp4">}}
 
+        <p class="figcaption">
+          Powered by <a href="https://jupyterbook.org"><img src="/images/logos/project/jupyterbook.svg" /></a> and <a href="https://mystmd.org"><img src="/images/logos/project/myst.svg" /></a>
+        </p>
```

Comment:
Thank you for the links to the underlying tools and communities when showcasing this content.